### PR TITLE
Add a configurable feedback URL to prototype

### PR DESCRIPTION
**Pull request by wchargin**
Summary:
We can now set, at build time, a URL to be displayed at the top of the
prototype, encouraging users to provide feedback. If the URL is not
provided, it defaults to the appropriate topic on the SourceCred
Discourse instance.

The result looks like this:

![Screenshot of the feedback URL in the prototype][screenshot]

[screenshot]: https://user-images.githubusercontent.com/4317806/44814824-a238b380-ab92-11e8-88c8-dfbae27ca496.png

Test Plan:
Unit tests added to `yarn sharness-full`.

You can run `yarn start` to see the message with the default URL, or
`SOURCECRED_FEEDBACK_URL=http://example.com/ yarn start` to specify a
custom URL.

wchargin-branch: feedback-url

### CODE REVIEW

File: src/app/credExplorer/App.js [hunk 1]
```diff
@@ -82,6 +82,14 @@ export function createApp(
       }
       return (
         <div style={{maxWidth: 900, margin: "0 auto", padding: "0 10px"}}>
+          <p>
+            <em>
+              This is a prototype. Please{" "}
+              <a href={process.env.SOURCECRED_FEEDBACK_URL}>
```
Comment (decentralion) [inline src/app/credExplorer/App.js]: Maybe add a test that the a has a href that is not null, just to verify there wasn't a typo or such

**Comment by decentralion** [comment]
Also, please make a note in the CHANGELOG
